lib/opentelemetry/trace/exporter/otlp.lua

```diff
@@ -7,13 +10,37 @@ local mt = {
     __index = _M
 }
 
-function _M.new(http_client)
+function _M.new(http_client, timeout_ms)
     local self = {
         client = http_client,
+        timeout_ms = timeout_ms or DEFAULT_TIMEOUT_MS,
```

Comment:
We might want to assert that the http client timeout is > the timeout_ms on the exporter.